client/extensions/woocommerce/state/sites/settings/email/reducer.js

```diff
@@ -0,0 +1,367 @@
+/**
+ * External dependencies
+ */
+import { keys } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { combineReducers } from 'state/utils';
+import {
+	WOOCOMMERCE_MAILCHIMP_CAMPAIGN_DEFAULTS_SUBMIT_SUCCESS,
```

Comment:
There has been some discussion around alphabetizing as of late, not sure if this import list should be alphabetized or not.